plugins/orchestrator/package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "@janus-idp/backstage-plugin-orchestrator",
+  "version": "0.0.1",
+  "license": "Apache-2.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "frontend-plugin"
+  },
+  "homepage": "https://janus-idp.io/",
+  "repository": "github:janus-idp/backstage-plugins",
+  "bugs": "https://github.com/janus-idp/backstage-plugins/issues",
+  "keywords": [
+    "backstage",
+    "plugin",
+    "orchestrator",
+    "workflows"
+  ],
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "start": "backstage-cli package start",
+    "build": "backstage-cli package build",
+    "tsc": "tsc",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test --passWithNoTests --coverage",
+    "clean": "backstage-cli package clean",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack"
+  },
+  "dependencies": {
+    "@backstage/core-components": "^0.13.6",
+    "@backstage/core-plugin-api": "^1.7.0",
+    "@backstage/errors": "^1.2.3",
+    "@backstage/plugin-scaffolder-common": "^1.4.2",
+    "@backstage/plugin-scaffolder-react": "^1.5.1",
+    "@backstage/types": "^1.1.1",
+    "@janus-idp/backstage-plugin-orchestrator-common": "0.0.1",
+    "@kie-tools-core/editor": "^0.32.0",
+    "@kie-tools-core/envelope-bus": "^0.32.0",
+    "@kie-tools-core/notifications": "^0.32.0",
+    "@kie-tools-core/react-hooks": "^0.32.0",
+    "@kie-tools/serverless-workflow-combined-editor": "^0.32.0",
+    "@kie-tools/serverless-workflow-language-service": "^0.32.0",
+    "@kie-tools/serverless-workflow-service-catalog": "^0.32.0",
+    "@material-ui/core": "^4.12.4",
+    "@material-ui/icons": "^4.11.3",
+    "@rjsf/core": "^3.2.1",
+    "@rjsf/core-v5": "npm:@rjsf/core@5.7.3",
+    "@rjsf/material-ui-v5": "npm:@rjsf/material-ui@5.7.3",
+    "@rjsf/utils": "5.7.3",
+    "@rjsf/validator-ajv8": "5.7.3",
+    "json-schema": "^0.4.0",
```

Comment:
Let's avoid this library

Reply:
I understand the concern but this lib is also used by Backstage, so at least we are in sync with them. We can re-evaluate replacing this library in a future opportunity.